tests/test_package.py

```diff
@@ -0,0 +1,68 @@
+import io
+import logging
+import os
+import shutil
+from contextlib import contextmanager
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from snowcli.cli.snowpark import package
+
+from tests.test_data import test_data
+
+
+class TestPackage:
+    LOGGER = logging.getLogger(__name__)
```

Comment:
it is not needed

Reply:
You`re right. It`s a sole survivor of a different approach :D